### PR TITLE
Fix templates for support to allow quotes, brackets, etc.

### PR DESCRIPTION
**Pull request by cdosborn**
TLDR: let the email ticket system be responsible for sanitizing support emails

By default django auto-escapes all templates. You have to mark something as
safe for it to be not escaped. Our issue is that django was escaping html,
and then RT escaped the escaped content!

**note:** these templates were tested!

### CODE REVIEW

File: core/templates/core/email/feedback.html [hunk 1]
```diff
@@ -10,16 +10,16 @@
 
 Instances :
 {% for i in instances %}
-    Name : {{ i.name }}
-    ID : {{ i.provider_alias }}
+    Name : {{ i.name|safe }}
+    UUID : {{ i.provider_alias }}
```
Comment (cdosborn) [inline core/templates/core/email/feedback.html]: Small name change. A model id mean something different that a models uuid

File: core/templates/core/email/volume_report.html [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
 {% endif %}
 Details : 
-    {{ message }}
+    {{ message|safe }}
 
 Volume :
-    Name : {{ volume.name }}
-    ID : {{ volume.id }}
+    Name : {{ volume.name|safe }}
+    UUID : {{ volume.identifier }}
```
Comment (cdosborn) [inline core/templates/core/email/volume_report.html]: We care about the volume uuid more than the model id

**Comment by steve-gregory** [comment]
@cdosborn should this be pointing to `petrified-penguin`?

**Comment by cdosborn** [comment]
@steve-gregory Somehow extra commits got tacked on here. I'll resubmit a pr against PP.

**Pull request closed without merging**